spt/OrangeBox/spt-serverplugin.cpp

```diff
@@ -506,6 +506,46 @@ CON_COMMAND(y_spt_cvar2, "CVar manipulation, sets the CVar value to the rest of
 }
 #endif
 
+void TestCanJb(float height)
+{
+	Vector player_origin = clientDLL.GetPlayerEyePos();
+	Vector vel = serverDLL.GetLastVelocity();
+
+	constexpr float gravity = 600;
+	constexpr float groundThreshold = 2.0f;
+	constexpr float frametime = 0.015f;
+	constexpr int maxIterations = 1000;
+
+	for (int i = 0; i < maxIterations && (vel.z > 0 || player_origin.z >= height); ++i)
+	{
+		if (vel.z < 0 && player_origin.z - height <= groundThreshold)
+		{
+			Msg("Yes, projected landing height %.6f\n", player_origin.z);
+			return;
+		}
+
+		vel.z -= (gravity * frametime / 2);
+		player_origin.z += vel.z * frametime;
+		vel.z -= (gravity * frametime / 2);
+	}
+	Msg("No\n");
```

Comment:
Not sure I'm a fan of this "No" message; maybe something more extended like "Jumpbug is impossible"? The yes one can stay the same.